Makefile

```diff
@@ -67,3 +69,7 @@ clean:                              ## Clean docker stuff
 
 env:                                ## Print current env
 	env | sort
+
+trivy: 								## Detect image vulnerabilities
+	$(dc) build app
```

Comment:
Ik zit ineens te bedenken dat wanneer je hier een build doet, en je de `apt install` dingen nog in je cache hebt staan, hij alsnog de oude packages neemt van destijds van je build. Als je hier nog een `--no-cache` aan toevoegt, dan weet je zeker dat je trivy check op de laatste versie van de apt packages wordt gedraaid.